Dart/src/com/jetbrains/lang/dart/ide/runner/server/vmService/DartVmServiceDebugProcess.java

```diff
@@ -279,14 +287,16 @@ public void guessRemoteProjectRoot(@NotNull final ElementList<LibraryRef> librar
 
         LOG.assertTrue(file.getPath().startsWith(localProjectRoot.getPath() + "/"), file.getPath() + "," + localProjectRoot.getPath());
         final String relPath = file.getPath().substring(localProjectRoot.getPath().length()); // starts with slash
-        if (!relPath.startsWith("/lib/") && remoteUri.endsWith(relPath)) {
+        if (remoteUri.endsWith(relPath)) {
           howManyFilesMatch++;
           myRemoteProjectRootUri = remoteUri.substring(0, remoteUri.length() - relPath.length());
         }
       }
 
       if (howManyFilesMatch == 1) {
         break; // we did the best guess we could
+      } else if (howManyFilesMatch > 0) {
+        // TODO: Look for an additional way to determine `myRemoteProjectRootUri`.
```

Comment:
I've removed TODO, if we do not break on `howManyFilesMatch == 1` then we try to improve our guess on next iteration (we are inside of a loop here).